nl/firefox-ios.xliff

```diff
@@ -1129,21 +1134,28 @@
         <target>U dient %@ te verifiëren. Controleer uw e-mail op de verificatiekoppeling van Firefox.</target>
         <note>Message explaining that user needs to check email for Firefox Account verfication link.</note>
       </trans-unit>
+      <trans-unit id="FxA.ConnectAnotherDevice">
+        <source>Connect Another Device</source>
+        <note>Button label to connect another device to Sync</note>
+      </trans-unit>
       <trans-unit id="FxA.FirefoxAccount">
         <source>Firefox Account</source>
         <target>Firefox-account</target>
         <note>Settings section title for Firefox Account</note>
       </trans-unit>
       <trans-unit id="FxA.ManageAccount">
-        <source>Manage Account</source>
-        <target>Account beheren</target>
+        <source>Manage Account &amp; Devices</source>
```

Comment:
@Delphine will this change show up in the web tools?

Reply:
Yes, because there's no target (it's removed by the update script)